catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/RemoteSolrCatalogProvider.java

```diff
@@ -47,6 +50,16 @@
 
   private static final Properties DESCRIBABLE_PROPERTIES = new Properties();
 
+  private FilterAdapter filterAdapter;
+
+  private SolrClient client;
+
+  private SolrFilterDelegateFactory solrFilterDelegateFactory;
+
+  protected DynamicSchemaResolver resolver;
```

Comment:
❓ Why does it have to be protected? Again opening up the implementation to outside changes.